.editorconfig

```diff
@@ -14,3 +14,6 @@ trim_trailing_whitespace = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[gradlew.bat]
+end_of_line = crlf
```

Comment:
I think you need to add it to `.gitattributes` as well

Reply:
True